src/router.js

```diff
@@ -46,13 +46,14 @@ const routes = [
   { path: '/challenges/:id/prices', name: 'challenge-prices', component: () => import('./views/ChallengePriceList.vue'), meta: { title: 'Challenge prices', breadcrumbs: [{title: 'Challenges', disabled: false, to: '/challenges' }, {title: 'ID', disabled: false, to: '/challenges/:id' }, {title: 'Prices', disabled: true }] }},
   { path: '/challenges/:id/proofs', name: 'challenge-proofs', component: () => import('./views/ChallengeProofList.vue'), meta: { title: 'Challenge proofs', breadcrumbs: [{title: 'Challenges', disabled: false, to: '/challenges' }, {title: 'ID', disabled: false, to: '/challenges/:id' }, {title: 'Proofs', disabled: true }] }},
   { path: '/experiments', name: 'experiments', component: () => import('./views/Experiments.vue'), meta: { title: 'Experiments', icon: 'mdi-test-tube', drawerMenu: true, drawerMenuConditionalDisplay: 'drawer_display_experiments', breadcrumbs: [{title: 'Experiments', disabled: true }] }},
-  { path: '/experiments/contribution-assistant', name: 'contribution-assistant', component: () => import('./views/ContributionAssistant.vue'), meta: { title: 'ContributionAssistant', icon: 'mdi-draw', requiresAuth: true, breadcrumbs: [{title: 'Experiments', disabled: false, to: '/experiments' }, {title: 'ContributionAssistant', disabled: true }] }},
+  { path: '/experiments/proof-price-tag-assistant', name: 'price-tag-assistant', component: () => import('./views/ProofPriceTagAssistant.vue'), meta: { title: 'ProofPriceTagAssistant', icon: 'mdi-draw', requiresAuth: true, breadcrumbs: [{title: 'Experiments', disabled: false, to: '/experiments' }, {title: 'ProofPriceTagAssistant', disabled: true }] }},
```

Comment:
made the corresponding change in the backend as well : https://github.com/openfoodfacts/open-prices/commit/50f1608f687e32dad81f594a6979e931b96107f0